internal/beater/server.go

```diff
@@ -191,14 +199,20 @@ func newServer(args ServerParams, listener net.Listener) (server, error) {
 		return server{}, err
 	}
 
-	otlpBatchProcessor := args.BatchProcessor
+	var otlpBatchProcessor modelprocessor.Chained
```

Comment:
`args.BatchProcessor` is now ignored, is that intended? I think this will cause everything to send to nowhere.

Reply:
This is a great point, I didn't notice it 😨 - I'll add it back.

Reply:
I've just re-added it